src/test/java/seedu/address/model/person/PersonWithCriteriaPredicateTest.java

```diff
@@ -102,6 +109,29 @@ public void test_personNotWithCriteria_returnsFalse() {
                 LocalDate.of(2024, 1, 1), LocalDate.of(2024, 12, 31)
         ));
         assertFalse(predicate.test(new PersonBuilder().withAge("19").addAppointment("01/01/2024 1200").build()));
+
```

Comment:
Can add more tests.

Reply:
I have pushed another commit to cover the path! Thanks for pointing out!